apps/admin-web/package.json

```diff
@@ -7,23 +7,38 @@
     "@auth0/auth0-spa-js": "^1.2.2",
     "antd": "^3.12.4",
     "apollo-boost": "^0.4.3",
+    "apollo-cache": "^1",
+    "apollo-client": "^2",
+    "apollo-link": "^1",
```

Comment:
Do you use these dependencies? I see them added, but not used

Reply:
I think since lerna is in here, no matter how many packages I remove from the required, it won't scream at me, so I am worried about it, but I will remove these.